model.schema.json

```diff
@@ -5,10 +5,62 @@
     "description": "A MetaStats object representation for statistical analysis",
     "type": "object",
     "properties": {
-        "qbicId": {
-            "description": "QBiC internal identifier for sample entities",
+        "samplePreparationId": {
+            "description": "QBiC internal identifier for preparation samples",
             "type": "string",
             "pattern": "Q[A-X0-9]{4}[0-9]{3}[A-X0-9]{2}"
+        },
+        "sequencingFacilityId": {
+            "description": "Name of sample given by the Sequencing Facility",
+            "type": "string"
+        },
+        "sampleName": {
+            "description": "ID of the sample written on the tube",
+            "type": "string"
+        },
+        "sampleGroup": {
+            "description": "Grouping of samples based on shared QBiC internal identifier for entity samples",
+            "type": "string",
+            "pattern": "Q[A-X0-9]{4}ENTITY-[1-9][0-9]*"
```

Comment:
openBIS individuum entities are not the factors

Reply:
Oh so now you added the vocabularies to the metadata-object-model directly. As this might get a bit messy, don't you think it would be a good idea to keep them in another separate json and import them here? (if that is possible)

Reply:
We decided to do it like this for the first release but we will move the definition since it makes it all a bit messy, you are right :)